cdap-api-common/src/main/java/co/cask/cdap/api/common/Bytes.java

```diff
@@ -257,6 +257,20 @@ public static String toHexString(byte [] bytes) {
   }
 
   /**
+   * Returns a byte[] by decoding from the given hexadecimal string. The string must be have an even length.
+   */
+  public static byte[] fromHexString(String str) {
```

Comment:
Funny. I know that when I added Bytes.stopKeyForPrefix(), I also added unit tests for that. Where did they go? Were they deleted when the class moved from common to api?